compute_tools/src/tls.rs

```diff
@@ -73,15 +75,18 @@ fn try_update_key_path_blocking(pg_data: &Path, tls_config: &TlsConfig) -> Resul
         .create(true)
         .truncate(true)
         .mode(0o600)
-        .open(pg_data.join(SERVER_KEY))?;
+        .open(postgresql_conf_path.join(SERVER_KEY))?;
```

Comment:
Can you help explain this to me?

It seems like your new paths would look like:

```
path/to/pgdata/postgresql.conf/server.key
```

Why is that correct?

Reply:
Ah - no I just confused myself reading the code. It's not correct, will revert.